core-helm-chart/values.yaml

```diff
@@ -16,20 +16,20 @@
 image:
   repository: quay.io/kubestellar/kubestellar
   pullPolicy: IfNotPresent
-  tag: release-0.14
+  tag: sp-mgt-pr2
 
 # Space abstraction layer image parameters
 spaceimage:
   repository: quay.io/kubestellar/space-framework
   pullPolicy: IfNotPresent
-  tag: sp-mgt
+  tag: sp-mgt-pr2
 
-# The namespace used by the space abstraction layer.  
-spaceNamespace: default
-
-# The space framework default provider
+# The space framework default provider type.
 # Possible values: [kcp|kubeflex]
-defaultSpaceProvider: kcp
+defaultSpaceProviderType: kcp
+
+# Spaces are located within the same cluster as the kubestellar pod.
```

Comment:
I see an unconditional assertion here. It is followed by a boolean variable set to true, which I assume is associated with the unconditional assertion here. What happens when that boolean variable is set to false?

Reply:
Is this better?  
```
# Spaces can be accessed internally (i.e. inCluster) or externally.
# true: spaces are accessed internally from a pod within the cluster that host them.
# false: spaces are accessed outside of the hosting cluster.
inClusterSpaces: true
```

Reply:
That passes syntax checking. But it does not make sense to me. Normal usage of the Helm chart will involve both sorts of access, not just one.

Reply:
Parameter removed.